src/utils/wallet.ts

```diff
@@ -475,13 +478,21 @@ const wallet = {
       };
     }
 
-    return {
+    const accessData: IWalletAccessData = {
       walletType,
       multisigData,
       mainKey: encryptedMainKey,
       xpubkey: xpriv.xpubkey,
       walletFlags: 0,
     };
+
+    if (acctXpriv !== null) {
```

Comment:
Should we also check for `undefined`?

Maybe `acctXpriv != null`

Reply:
The variable is initialized as null and bitcore-libs derivation does not return undefined.